[2024-03-18 23:56:36,633] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: opcion_2.extraer_datos2 scheduled__2024-03-17T22:36:15.609958+00:00 [queued]>
[2024-03-18 23:56:36,641] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: opcion_2.extraer_datos2 scheduled__2024-03-17T22:36:15.609958+00:00 [queued]>
[2024-03-18 23:56:36,642] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-03-18 23:56:36,643] {taskinstance.py:1377} INFO - Starting attempt 2 of 3
[2024-03-18 23:56:36,644] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-03-18 23:56:36,657] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extraer_datos2> on 2024-03-17 22:36:15.609958+00:00
[2024-03-18 23:56:36,661] {standard_task_runner.py:52} INFO - Started process 375 to run task
[2024-03-18 23:56:36,664] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'opcion_2', 'extraer_datos2', 'scheduled__2024-03-17T22:36:15.609958+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/tareas2.py', '--cfg-path', '/tmp/tmpl_9z4ju2', '--error-file', '/tmp/tmpzl6ibq77']
[2024-03-18 23:56:36,666] {standard_task_runner.py:80} INFO - Job 30: Subtask extraer_datos2
[2024-03-18 23:56:36,724] {task_command.py:371} INFO - Running <TaskInstance: opcion_2.extraer_datos2 scheduled__2024-03-17T22:36:15.609958+00:00 [running]> on host 1ce5256cfda3
[2024-03-18 23:56:36,789] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=opcion_2
AIRFLOW_CTX_TASK_ID=extraer_datos2
AIRFLOW_CTX_EXECUTION_DATE=2024-03-17T22:36:15.609958+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-17T22:36:15.609958+00:00
[2024-03-18 23:56:38,277] {python.py:173} INFO - Done. Returned value was: None
[2024-03-18 23:56:38,293] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=opcion_2, task_id=extraer_datos2, execution_date=20240317T223615, start_date=20240318T235636, end_date=20240318T235638
[2024-03-18 23:56:38,331] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-03-18 23:56:38,365] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
